tcms/testruns/tests/test_views.py

```diff
@@ -94,24 +94,24 @@ def test_refuse_if_missing_plan_pk(self):
         self.client.login(  # nosec:B106:hardcoded_password_funcarg
             username=self.tester.username,
             password='password')
-        response = self.client.post(self.url, {})
+        response = self.client.get(self.url, {})
         self.assertRedirects(response, reverse('plans-search'))
 
     def test_refuse_if_missing_cases_pks(self):
         self.client.login(  # nosec:B106:hardcoded_password_funcarg
             username=self.tester.username,
             password='password')
-        response = self.client.post(self.url, {'from_plan': self.plan.pk}, follow=True)
+        response = self.client.get(self.url, {'p': self.plan.pk}, follow=True)
         self.assertContains(response, _('Creating a TestRun requires at least one TestCase'))
 
     def test_show_create_new_run_page(self):
```

Comment:
name here is inappropriate after the changes in behavior. Use `test_get_shows_selected_cases`